utils/node_local_test.py

```diff
@@ -2035,7 +2021,7 @@ def test_rename_clobber(self):
 
         # Create all three files in the dfuse instance we're checking.
         for index in range(3):
-            fd = open(os.path.join(self.dfuse.dir, 'file.{}'.format(index)), 'w')
+            fd = open(join(self.dfuse.dir, 'file.{}'.format(index)), 'w')
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations